redbot/cogs/bank/bank.py

```diff
@@ -124,4 +124,13 @@ def __init__(self, bot: Red):
         await bank.set_currency_name(name, ctx.guild)
         await ctx.send(_("Currency name has been set to: {name}").format(name=name))
 
+    async def on_command_error(self, ctx, exception):
+        author = ctx.author
+        command = ctx.command.qualified_name
+        data = await bank._conf.member(author).last_command_cost()
```

Comment:
This isn't a safe method of handling. async code makes no guarantees about the order this is handled compared to what another command does.